### DISCUSSION TITLE
Welcome!

### DISCUSSION THREAD

**Discussion by zsunberg** [JuliaPOMDP/POMDPs.jl#330] [General]
Welcome to the POMDPs.jl discussions page! Please feel free to post any type of question regarding POMDPs.jl by clicking on the New Discussion button in the Discussions tab. We will do our best to answer promptly!

**Comment by logankilpatrick** [comment]
What is a POMDP? Asking for a friend at NASA...

**Reply by mykelk** [reply]
Life. ;-)

![20151023_0050](https://user-images.githubusercontent.com/6294079/102904987-5d9e4980-4427-11eb-8955-23a7e38550c6.JPG)

If you're interested in learning the mathematics involved, you might want to check out [this book](http://algorithmsbook.com/).
